### PR TITLE
Demonstrate kw_args usage for and_wrap_original

### PR DESCRIPTION
**Pull request by JonRowe**

### CODE REVIEW

**Review by pirj — APPROVED** [review]
Perfect, thank you!

File: features/support/env.rb [hunk 1]
```diff
@@ -20,3 +20,10 @@
     set_environment_variable('RBXOPT', "-Xint=true #{ENV['RBXOPT']}")
   end
 end
+
+Before('@kw-arguments') do
+  unless RSpec::Support::RubyFeatures.kw_args_supported?
```
Comment (pirj) [inline features/support/env.rb:25]: Wondering if `require 'rspec/support/ruby_features'` is needed here like we do e.g. [here](https://github.com/rspec/rspec-core/blob/640a9f62d954f69993b9e95511303dbda9b085e2/features/support/jruby.rb#L2).

Comment (JonRowe) [reply features/support/env.rb:25]: Its the skip logic thats broken for some reason

Comment (JonRowe) [reply features/support/env.rb:25]: I think its probably the ancient version of cucumber that the legacy specs run :joy:

File: features/configuring_responses/wrapping_the_original_implementation.feature [hunk 2]
```diff
@@ -51,3 +51,31 @@ Feature: Wrapping the original implementation
       """
     When I run `rspec spec/and_wrap_original_spec.rb`
     Then the examples should all pass
+
+  @kw-arguments
```
Comment (pirj) [inline features/configuring_responses/wrapping_the_original_implementation.feature:55]:
```suggestion
  @required-kwargs
```

File: features/support/env.rb [hunk 3]
```diff
@@ -20,3 +20,14 @@
     set_environment_variable('RBXOPT', "-Xint=true #{ENV['RBXOPT']}")
   end
 end
+
+Before('@kw-arguments') do |scenario|
+  unless RSpec::Support::RubyFeatures.kw_args_supported?
```
Comment (pirj) [inline features/support/env.rb:25]:
```suggestion
Before('@required-kwargs') do |scenario|
  unless RSpec::Support::RubyFeatures.required_kw_args_supported?
```

File: script/functions.sh [hunk 4]
```diff
@@ -63,6 +63,11 @@ function run_cukes {
       RUBYOPT="-I${PWD}/../bundle -rbundler/setup" \
          PATH="${PWD}/bin:$PATH" \
          bin/cucumber --strict
+    elif is_mri_2; then
+      RUBYOPT="${RUBYOPT} -I${PWD}/../bundle -rbundler/setup" \
+         PATH="${PWD}/bin:$PATH" \
+         bin/cucumber --strict --format pretty features/configuring_responses/wrapping_the_original_implementation.feature:80
```
Comment (pirj) [inline script/functions.sh:69]: I love how `--format pretty` immediately showed the underlying failure message!

Comment (JonRowe) [reply script/functions.sh]: It actually didn't I had to add a false statement to get the spec output

File: features/support/env.rb (hunk 3 above)
Comment (JonRowe) [reply features/support/env.rb:25]: I fixed the implementation instead.

**Review by pirj — APPROVED** [review]
Awesome!

**Pull request merged**